packages/ui/src/components/account/AccountMenu.tsx

```diff
@@ -65,14 +71,14 @@ const AccountMenu = () => {
         disabledOptions[exportAccountDataLabel] = true
         tooltipOptions[exportAccountDataLabel] =
             "Not available for Hardware Wallets accounts."
-        options.push({
+        options.splice(options.length-1, 0, {
```

Comment:
I would add an "order" property here and sorting the array before rendering.

Reply:
The thing is, we don't need the order as we only require the **switch accounts btn** to be the **last** and will have to handle many border cases where a dev might add two btns with the same order number, or we at least need to know the number of btns in array every time we include a new btn so that the numbers are correct.
Let me know what you think🙏🏻

Reply:
If we need to set the switch account button to the be last I would simply add an options.push right after the if. The point is that with this implementation you don't ensure the `switch account` option to be the last if someone else adds a new entry to the options array. Do I explain myself?

Reply:
Yeah got it!
Thanks man 💪🏼